src/patternfly/base/_variables.scss

```diff
@@ -12,3 +12,35 @@
 @include pf-v6-rtl {
   @include pf-v6-set-inverse;
 }
+
+// stylelint-disable
+.pf-m-no-motion * {
+  // simplest way of turning off all animation
+  // disadvantage: takes the wildcard selector
+  // transition: none !important;
+  // animation: none !important;
+}
+// styleline-enable
+
+.pf-m-no-motion {
+  // instead we can set all (or just semantic?) motion tokens to 0
+  // disadvantage - need to catch any new tokens created
+  // note - relies on always using tokens
+  --pf-t--global--delay--400: 0s;
+  --pf-t--global--delay--300: 0s;
+  --pf-t--global--delay--200: 0s;
+  --pf-t--global--delay--100: 0s;
+  --pf-t--global--duration--300: 0s;
+  --pf-t--global--duration--200: 0s;
+  --pf-t--global--duration--100: 0s;
+  --pf-t--global--duration--000: 0s;
+  --pf-t--global--motion--delay--long: 0s;
+  --pf-t--global--motion--delay--default: 0s;
+  --pf-t--global--motion--delay--short: 0s;
+  --pf-t--global--motion--delay--none: 0s;
+  --pf-t--global--motion--duration--long: 0s;
+  --pf-t--global--motion--duration--default: 0s;
+  --pf-t--global--motion--duration--short: 0s;
+  // added just for testing purposes
+  --pf-t--global--motion--duration--test: 0s;
+}
```

Comment:
@mattnolting @mcoker I'd be interested in your thoughts about the two methods of turning off all motion here. (this isn't reduced motion - it's turning everything off for automated testing purposes)

Reply:
In [#6369](https://github.com/patternfly/patternfly/issues/6369#issue-2160813742), Garrett notes performance impacts of using global selectors. At the least, I'd say `> *` is nonviable. Second approach 👍 

> You can see it uses > * in a few places (as well as :first-child, which isn't nearly as bad, but still not ideal). As it's [children of the tr, those can only be th and td according to tr's content model](https://html.spec.whatwg.org/multipage/tables.html#the-tr-element) (except for non-visible elements like script and template, which CSS doesn't need to touch). Even :where(th, td) should be much better. This is because [CSS matches from the right to the left](https://stackoverflow.com/questions/5797014/why-do-browsers-match-css-selectors-from-right-to-left/5813672#5813672), so those lines are saying "look at every element on the page (including children and parents of the wanted elements as well as everything else), then see if it has a parent of a th" instead of first narrowing it down to td and th elements.